rest/utilities_testing.go

```diff
@@ -1062,11 +1138,17 @@ func (bt BlipTester) DatabaseContext() *db.DatabaseContext {
 	return nil
 }
 
+// getBlipTesterSpec returns a default tester specification.
+func getDefaultBlipTesterSpec() BlipTesterSpec {
+	return BlipTesterSpec{GuestEnabled: true}
+}
+
+// NewBlipTesterFromSpecWithRT creates a blip tester from an existing rest tester
 func NewBlipTesterFromSpecWithRT(tb testing.TB, spec *BlipTesterSpec, rt *RestTester) (blipTester *BlipTester, err error) {
 	blipTesterSpec := spec
 	if spec == nil {
-		// Default spec
-		blipTesterSpec = &BlipTesterSpec{}
+		spec := getDefaultBlipTesterSpec()
+		blipTesterSpec = &spec
```

Comment:
It looks like this didn't previously set GuestEnabled?  I'm not even sure why that would be the default, but we should at minimum not change previous behaviour here.

Reply:
Without setting `GuestEnabled`, blip tester will fail to run correctly - the server fails with a handshake error. I don't think this code is executing, but it's out of scope to modify here.